src/AppBundle/Resources/translations/home.en.yml

```diff
@@ -15,60 +15,61 @@ whatyouneedListItem1: your case number (look for 'OPG reference' on letters from
 
 provideDetails:
-  heading: "You'll need to provide details of:"
+  summary: What you'll need for the report
+  para01: "You'll need to provide details of:"
   listItem01: how the client is cared for
   listItem02: purchases you've made on the client's behalf
   listItem03: financial records relating to the client (for example bank statements)
   listItem04: the client's assets and debts
   listItem05: any big decisions you have made or are planning to make for the client
   listItem06: friends, family and professionals you've contacted on behalf of the client
 
-hiddenText01:
-  summary: The client has more than one deputy
-  content:
-    para01: |
-      You'll only need to submit one deputy report for each reporting period. This applies 
-      if you make decisions together (usually called 'jointly') or separately and together 
-      (usually called 'jointly and severally').
+moreThanOne:
+  summary: What if there's more than one deputy?
+  para01: |
+    You'll only need to submit one deputy report for each reporting period. This applies
+    if you make decisions together (usually called 'jointly') or separately and together
+    (usually called 'jointly and severally').
 
-hiddenText02:
+afterISubmit:
   summary: What happens after I submit my report?
-  content:
-    para01: |
-      We'll review the report, and may still need to contact you for more information – by 
-      phone, email, in writing or by arranging a visit.
+  para01: |
+    We'll review the report, and may still need to contact you for more information – by
+    phone, email, in writing or by arranging a visit.
 
-hiddenText03:
-  summary: Health and welfare deputies
-  content:
-    para01: |
-      We are currently building a digital service for health and welfare deputies. In the 
-      meantime, please <a href="https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/537794/OPG104-Deputy-report-form-health-and-welfare-decisions.pdf" target="_blank">
-      download and print</a> your annual report form.
+hwDeputies:
+  summary: I'm a health and welfare deputy
+  para01: |
+    We are currently building a digital service for health and welfare deputies. In the
+    meantime, please <a href="https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/537794/OPG104-Deputy-report-form-health-and-welfare-decisions.pdf" target="_blank">
+    download and print</a> your annual report form.
 
-hiddenText04:
-  summary: Deputy fees remission or exemption
-  content:
-    para01: You pay OPG's fees for supervising your deputyship from the client's funds.
-    para02: |
-      Clients with a lower income may qualify for a <a href="https://www.gov.uk/government/publications/deputy-fees-remission-or-exemption" target="_blank">
-      remission or exemption</a>.
+relatedLinks:
+  heading: "Deputies: make decisions for someone who lacks capacity"
+  link01Text: "Deputy guidance: how to carry out your duties"
```

Comment:
@Biggsen  you could also use a syntax like
link01:
 text: ...
 src: ...

then use with {{  link01.text }} and {{  link01.src }}